src/Effect/ReadFS.hs

```diff
@@ -38,17 +38,37 @@ module Effect.ReadFS (
   module X,
 ) where
 
-import Control.Algebra as X
-import Control.Carrier.Simple
-import Control.Effect.Diagnostics
+import Control.Algebra as X (
+  Algebra (alg),
+  Handler,
+  Has,
+  run,
+  send,
+  thread,
+  (~<~),
+  type (:+:) (L, R),
+ )
```

Comment:
Also, these were weirdly hard to fix.  When we re-export, we have to check the module that imports this to see if there's an issue.